frontend/packages/ux-editor/src/components/config/expressions/DynamicContent.tsx

```diff
@@ -18,19 +20,31 @@ import classes from './DynamicContent.module.css';
 interface ExpressionProps {
   component: FormComponent | FormContainer;
   dynamic: Dynamic;
-  onGetProperties: (dynamic: Dynamic) => {availableProperties: string[], expressionProperties: string[]}; // actions?
+  onGetProperties: (dynamic: Dynamic) => { availableProperties: string[], expressionProperties: string[] }; // actions?
```

Comment:
Should we just remove that comment? 🙈 
```suggestion
  onGetProperties: (dynamic: Dynamic) => { availableProperties: string[], expressionProperties: string[] };
```

Reply:
Yes, we can! If you feel property is an okay formulation? Because the main issue #8771 uses action 🤔